examples/API_v2/recent_tweets_count.py

```diff
@@ -0,0 +1,18 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace with your own search query
+query = 'covid -is:retweet'
```

Comment:
A more on-topic query like "Tweepy" or "Twitter API" might be better here.

Reply:
Updated to use tweepy